eosc/cmd/voteRemoveAll.go

```diff
@@ -0,0 +1,31 @@
+package cmd
+
+import (
+	"github.com/eoscanada/eos-go"
+	"github.com/eoscanada/eos-go/system"
+	"github.com/spf13/cobra"
+)
+
+var voteRemoveAllCmd = &cobra.Command{
+	Use:   "remove-all [voter name]",
+	Short: "Remove all votes currently casted - effectively voting for no one.",
+	Args:  cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		api := getAPI()
+
+		voterName := toAccount(args[0], "voter name")
+
+		var noVotes []eos.AccountName
+		pushEOSCActions(api,
+			system.NewVoteProducer(
```

Comment:
Same thing here, should we have some kind of checks for `proxy`? This does not seem to handle the proxy case.